Update translations

cicnavi opened the pull request:
I've gone through translations for existing languages and updated them. I've used Google Translate so I expect it to have (a lot of) inadequate or too literal translations. But since we have those languages from the beginning, I thought it might be valuable to have them even in that form...

tvdijen commented:
I'm not sure this is a good idea.. The Dutch translations were pretty bad.. Some technical phrases are better left in English because they just don't translate well.

cicnavi commented:
Thanks for the input, I was afraid of that... So this presents maintenance burden, and also I don't really know which translation is ok and which is not ok... If I leave only old translations (and fallback to en for new ones), only like 10 % will be actually translated... So, I'm better of removing those languages totally. If someone wants them, we can add them at any time.

tvdijen commented:
Nah, we just gotta ask on Slack if people can help us with the translations.. I think we have all nationalities covered.  Jaime can do Spanish, I believe Stefan speaks French.. And I'm pretty sure we have Italians there too.

cicnavi commented:
I've deleted usupported langs, and added Croatian.

A bit strange thing is happending now. When I select croatian lang (hrvatski) and if I go to 'Configuration' (Konfiguracija in hr), I see the menu item translated to 'Konfiguracija'. When I go to 'OIDC', the menu changes back to 'Configuration'.

![image](https://github.com/user-attachments/assets/37a08e1e-3e82-4498-b025-8e9f5c967e6f)

![image](https://github.com/user-attachments/assets/98db2e5d-f0be-4f5d-879f-a9111b3144d9)

Also, in OIDC sub-menu, I don't get any Croatian translations...

When I select language Netherlands, I don't see any translations (that is, only just a few translations). For example, Federation Settings should be translated...

![image](https://github.com/user-attachments/assets/7b3005af-3fab-4deb-b4a7-ac8f06dd87e7)

Is the same with you or this is only on my instalation?

cicnavi commented:
> Nah, we just gotta ask on Slack if people can help us with the translations.. I think we have all nationalities covered. Jaime can do Spanish, I believe Stefan speaks French.. And I'm pretty sure we have Italians there too.

Sorry, I only now see your comment, but either way I would be surprised if someone really tackles with that...

cicnavi commented:
I now see all translations as expected... Previously I've messed up my installation...

cicnavi commented:
I've added back es, fr and it langs, but as empty translations, so if you ever decide to call for help on those translations, they will have their placeholders...
